Automattic/jetpack
# VideoPress: filter videos by uploader

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: filter videos by uploader

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to VideoPress dashboard
* Open the filter
* Confirm you see the uploaders list there
* Confirm it's possible to filter videos by clicking on the Uploader


https://user-images.githubusercontent.com/77539/195888237-32d91075-4536-4bc3-98cd-8793b10dbd80.mov
